shuttle/templates/driver_check_in/load_driver_check_locations.html

```diff
@@ -61,6 +67,13 @@ <h4>Current status:<span class="badge badge-light break-badge">{{ current_break_
     </div>
 
     <script>
+        setInterval(check_time, 30000);
+            function check_time(){
+                var today = new Date();
+                var time = today.getHours() + ":" + today.getMinutes();
+                if(time >= {{ next_time.split(' ')[0]|tojson }}){
+                    location.reload();
+            }
```

Comment:
This can all be "outdented" since it should be on the same level as setInterval I believe